### PR TITLE
create releases on `v*.*.*` tags

### PR DESCRIPTION
**Pull request by adbenitez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,6 +212,11 @@ and activate it with `. env/bin/activate`. Afterwards run `pytest`.
 
 ### Releasing
 
-TODO: create the tarball and a GitHub release on version tags.
+To automatically build and create a new GitHub release:
+
+```
+git tag v1.0.1
```
[the remark this answers is not in the document]
Comment (adbenitez) [reply README.md]: updated, thanks!!! :+1:

**Review by Septias — CHANGES_REQUESTED** [review]
Can you depend on `build_dist` instead of building the tarball from scratch?

**Comment by adbenitez** [comment]
> Can you depend on `build_dist` instead of building the tarball from scratch?

this was easier for me, but feel free to push improvements to this PR

**Review by Septias — APPROVED** [review]

**Comment by Septias** [comment]
let's just merge this, I can make improvements later if it annoys me too much

**Comment by r10s** [comment]
> let's just merge this, I can make improvements later if it annoys me too much

i would not put too much effort in that :)

we still still aim to this repository to codeberg, so any effort wrt ci is better done there

**Comment by adbenitez** [comment]
> EDIT: /me speaking about CI only. just noticing that, other that the title implies, this pr is doing tons of other things :)

that was me trying to rebase hehe



> i would not put too much effort in that :)

at this point there is no effort other than pressing the merge button

**Comment by r10s** [comment]
> at this point there is no effort other than pressing the merge button

sure, all fine. i was referring to "I can make improvements later" :)

**Pull request merged**